examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -241,15 +207,18 @@ def evaluate(model, X, y, cv):
     )
 
 
-evaluate(gbrt_pipeline, X, y, cv=ts_cv)
+evaluate(gbrt, X, y, cv=ts_cv)
 
 # %%
 # This model has an average error around 4 to 5% of the maximum demand. This is
-# quite good for a first trial without any hyper-parameter tuning! We just had
-# to make the categorical variables explicit. Note that the time related
-# features are passed as is, i.e. without processing them. But this is not much
-# of a problem for tree-based models as they can learn a non-monotonic
-# relationship between ordinal input features and the target.
+# quite good for a first trial without any feature engineering nor
+# hyper-parameter tuning! We just had to make the categorical variables
+# explicit.
```

Comment:
```suggestion
# quite good for a first trial without any feature engineering,
# hyper-parameter tuning, or having to configure
# features as being categorical.
```

We just called `HistGradientBoostingRegressor()` so it is all "out of the box".

The caveat is of course that the pandas DF already contained the information. So there isn't any magic/auto-detection going on.

Reply:
We still need to make the categorical variables explicit with proper typing in the pandas dataframe.

Reply:
As far as I can tell in this example the data frame already comes with the correct dtype. So I think the "we just had to make the categorical variables explicit." is confusing. At least it makes me think that somewhere in this example we did that. Maybe we should just point out that the data frame already contains features that are marked as "category" via their dtype and hence everything just works (no user action is required).

Reply:
Just saw that we already do that (point out the dtypes) when loading the data.